8Knot/pages/repo_overview/visualizations/ossf_scorecard.py

```diff
@@ -40,6 +41,14 @@
                     [
                         dbc.Row(
                             [
+                                dbc.Col(
+                                    dbc.Row(
+                                        [
+                                            dbc.Label("Last Updated:", className="mr-2"),
+                                            html.Div(id=f"{PAGE}-{VIZ_ID}-updated"),
```

Comment:
put date next to "last updated" instead of underneath

Reply:
fixed

Reply:
add space after the :

Reply:
spaces added